cmd/gsw_service.go

```diff
@@ -25,41 +27,39 @@ func printTelemetryPackets() {
 			for _, measurementName := range packet.Measurements {
 				measurement, ok := proc.GswConfig.Measurements[measurementName]
 				if !ok {
-					fmt.Printf("\t\tMeasurement '%s' not found\n", measurementName)
+					logger.Warn(fmt.Sprint("Measurement '",measurementName,"' not found"))
 					continue
 				}
 				fmt.Printf("\t\t%s\n", measurement.String())
 			}
 		} else {
-			fmt.Println("\t\tNo measurements defined.")
+			logger.Warn("No measurement defined.")
 		}
 	}
 }
 
 func vcmInitialize(config *viper.Viper) (*ipc.IpcShmHandler, error) {
 	if !config.IsSet("telemetry_config") {
 		err := errors.New("Error: Telemetry config filepath is not set in GSW config.")
-		fmt.Printf("%v\n", err)
-		return nil, err
+		logger.Error(fmt.Sprint(err))
 	}
 	data, err := os.ReadFile(config.GetString("telemetry_config"))
 	if err != nil {
-		fmt.Printf("Error reading YAML file: %v\n", err)
-		return nil, err
+		logger.Error(fmt.Sprint("Error reading YAML file: ", err))
```

Comment:
Should still return the err out of there. Need to exit the function early

Reply:
This doesn't look resolved? GitHub still shows that you don't return early after this log statement